.github/workflows/scheduled_release.yml

```
@@ -0,0 +1,137 @@
+    publish:
+        needs: release
+        runs-on: ubuntu-latest
+        environment:
+            name: pypi
+        permissions:
+            id-token: write
+            contents: read
+        steps:
+            - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6
+              with:
+                  ref: ${{ needs.create_tag.outputs.new_tag }}
```

actionlint (1.7.12)

[error] property "create_tag" is not defined in object type {release: {outputs: {}; result: string}}

(expression)